AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet1Solutions.lean

```diff
@@ -0,0 +1,78 @@
+/-
+This is a solutions sheet.
+-/
+
+import Mathlib.Tactic
+
+/-
+Welcome!
+
+This is an interactive problem set, where you'll learn to use Lean 4,
+a theorem prover and programming language, while exploring the basics
+of abstract algebra. Through exercises on groups, rings, and fields, you'll
+formalize and verify mathematical proofs. By the end, you should be comfortable
+with using Lean 4 to prove and verify theorems.
+
+This problem set is written under the assumption that you, the user, have
+already completed the Natural Number Game. Additionally, it is assumed that
+the user is in the process of taking an abstract algebra class (i.e. 401 or 501),
+so explanations of concepts will be more concise than a textbook's.
+
+There are solutions provided for your reference. Should you be interested in
+learning more about specific tactics, Lean 4 documentation is available online.
+
+****
+
+Before we dive into abstract algebra in Section 1, we will begin with the basics
+of Lean in Section 0. As you saw in the Natural Number Game, in Lean, exercises
+are completed using tactics. Section 0 will familiarize you with a few basic tactics
+through some exercises concerning logic, functions, and sets.
+
+To get you used to reading Lean, let us begin with a very basic exercise from
+the Natural Number Game.
```

Comment:
Maybe mention that if the user is very familiar with lean, then they can skip to the next section

Reply:
Done!